packages/core/src/features/st-import.ts

```diff
@@ -183,8 +185,8 @@ function addImportSymbols(importDef: Imported, context: FeatureContext, dirConte
     });
     // import as typed symbol
     for (const [type, handler] of ImportTypeHook.entries()) {
-        if (type in importDef) {
-            for (const [localName, importName] of Object.entries(importDef[type])) {
+        if (type in importDef.typed) {
+            for (const [localName, importName] of Object.entries(importDef.typed[type])) {
                 handler(context, localName, importName, importDef);
             }
         }
```

Comment:
maybe the warning of invalid typed import should be in here

Reply:
done